### PR TITLE
Stack runfiles

### PR DESCRIPTION
**Pull request by mirzaees**
Proposed changes:

- make a separate run file for some functions like geo2rdr and resample to better allocate memory and 
   walltime when submitting as a job
- add a new product fine.unw to remove the effect of filtering after unwrap

### CODE REVIEW

**Review by piyushrpt — APPROVED** [review]

**Comment by piyushrpt** [comment]
@dbekaert and @hfattahi - when you find time, can you review this?

**Review by hfattahi — COMMENTED** [review]
> @dbekaert and @hfattahi - when you find time, can you review this?

I don't see my requested changes addressed. In particular rmFilter argument should not be hardcoded in Stack.py.

File: contrib/stack/topsStack/unwrap.py [hunk 1]
```diff
@@ -276,6 +281,53 @@ def runUnwrapIcu(infile, outfile):
     #unwImage.finalizeImage()
     unwImage.renderHdr()
 
+
+def remove_filter(intfile, filtfile, unwfile):
```
[the remark this answers is not in the document]
Comment (mirzaees) [reply contrib/stack/topsStack/unwrap.py:285]: Dear @hfattahi , please check again. I made those requested modifications

**Review by hfattahi — APPROVED** [review]
Thank you @mirzaees. This is a great feature and allows evaluating the impact of filtering on higher leveel time-series.

**Pull request merged**